internal/channelz/socket.go

```diff
@@ -70,13 +70,23 @@ type EphemeralSocketMetrics struct {
 	RemoteFlowControlWindow int64
 }
 
+// SocketType represents the type of socket, indicating whether it is a
+// NormalSocket or a ListenSocket.
```

Comment:
No need to enumerate the types of sockets in this comment.

Reply:
Removed